lib/rules/space-infix-ops.js

```diff
@@ -8,6 +8,23 @@
 // Rule Definition
 //------------------------------------------------------------------------------
 
+const OPERATORS = [
```

Comment:
is there any way this list could be imported from espree, so that as new operators are added, it doesn't go out of date?